interpreter/wasm.opam

```diff
@@ -0,0 +1,22 @@
+# This file is generated by dune, edit dune-project instead
```

Comment:
If this file is generated, is there a reason to check it in?

Reply:
Yes, that's standard. The main reason is that, you usually want to be able to run things like: `opam pin git@github.com:WebAssembly/spec.git`. But opam doesn't know that the package is to be built with `dune` and can't call `dune build wasm.opam`, thus we have to commit the file. There are plans to fix this but I don't know if it'll happen soon.

Reply:
It's not a bug, so no plans to fix it :-)

Checking in the `opam` file ensures that the package manager can just inspect that file from the release archive to know what to do with it.  If code needs to be run to generate the file, and that fails or otherwise has complex constraints (e.g. a different dune), then figuring out an installation plan becomes quite complex.

So this looks fine the way it is...

Reply:
I see. Somehow you folks must have a higher itch threshold against cluttered directories than I... ;)

Reply:
> we might even skip the generation step altogether by having opam invoke dune to get the generated opam file directly

From [discuss]. But I guess the plans changed one more time. ;-)